src/daemon/auth/authorizer.rs

```diff
@@ -137,7 +138,10 @@ impl Authorizer {
             Ok(None) => self.actor_from_def(ACTOR_DEF_ANON),
 
             // error during authentication
-            Err(err) => self.actor_from_def(ACTOR_DEF_ANON.with_auth_error(err.to_string())),
+            Err(err) => {
+                // reveives a commons::error::Error, but we need an ApiAuthError
+                self.actor_from_def(ACTOR_DEF_ANON.with_auth_error(ApiAuthError::from(err)))
```

Comment:
What about using into() here?

And while you could fix "reveives" I suggest instead just dropping the comment entirely and make the block consistent with the others.

So something like this:

```
Err(err) => self.actor_from_def(ACTOR_DEF_ANON.with_auth_error(err.into()))
```

Reply:
Crossed my mind, let's see if I can make that work. 031f5ce handles it. Note that I had to pull in the `crate::commons::` and `crate::daemons::` stuff into focus for all features. Either that or have an explicit `crate::commons::error::Error` as argument type in `with_auth_error`